components/post_markdown/post_markdown.tsx

```diff
@@ -114,6 +119,7 @@ export default class PostMarkdown extends React.PureComponent<Props> {
                 imagesMetadata={this.props.post?.metadata?.images}
                 postId={this.props.post?.id}
                 editedAt={this.props.post?.edit_at}
+                showPostEditedIndicator={this.props.showPostEditedIndicator}
```

Comment:
Instead of passing this any further, could we simplify this by just not passing `editedAt` when `showPostEditedIndicator` is false?

Reply:
Good point. Updated as suggested.